src/components/form/examples/props-factory-form.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component, h, State } from '@stencil/core';
+import { schema } from './props-factory-schema';
+
+/**
+ * Using `propsFactory`
+ *
+ * @link props-factory-schema.ts
+ * @link props-factory-picker.tsx
+ */
+@Component({
+    tag: 'limel-example-props-factory-form',
+    shadow: true,
+})
+export class PropsFactoryFormExample {
+    @State()
+    private formData: object = {
+        hero: 1001,
+    };
+
+    private somePropertyToInject = {
+        message: 'Success!',
+    };
```

Comment:
Is there a reason to why this is nested inside an object? It might just make this look more complicated than it is :thinking:  If a message needs to be injected into the component, a property called `message` is probably better. What do you think? :slightly_smiling_face: Same reasoning for the property in the other component

Reply:
Yeah, but this isn't supposed to just be a message, it's a placeholder for some complicated object that needs inserting, like `platform`…

Maybe we should change the names to indicate that more clearly? Or give it some more dummy properties or something? I'm not sure…

Reply:
You can inject whatever you want, either a simple message or a complicated object. I guess the intended use is to put properties here that cannot be expressed in the schema :thinking: The most simple thing here would probably be an object that is an instance of a class or just an object with a method.

But yeah, it's probably just a naming issue. If we can come up with something more realistic I think it would be great :slightly_smiling_face: